woocommerce/woocommerce-android
# IPP feedback banner business logic

#### Business logic for IPP feedback survey banner

**Note:** This PR is part of the IPP in-app feedback banner project https://github.com/woocommerce/woocommerce-android/issues/8116. It targets the feature branch feature/8116-ipp-feedback-banner.

### Description


This PR introduces two use case classes containing the business logic of the IPP feedback banner's behavior.

* `ShouldShowFeedbackBanner` contains logic deciding if the banner should be shown.
* `GetIPPFeedbackSurveyUrl` returns the survey URL dedicated to the current user.

In the next PR, I'll add the next use cases responsible for storing banner interactions (actions) in SharedPreferences.

### Testing instructions


There is no UI part associated with the use cases yet, so I just recommend to review unit tests: `ShouldShowFeedbackBannerTest` and `GetIPPFeedbackSurveyUrlTest`, and make sure they are complete.

### Images/gif
